ext/legacycustomsearches/CRM/Contact/Form/Search/Custom/FullText.php

```diff
@@ -444,13 +444,7 @@ public function count() {
    */
   public function contactIDs($offset = 0, $rowcount = 0, $sort = NULL, $returnSQL = FALSE) {
     $this->initialize();
-
-    if ($returnSQL) {
-      return $this->all($offset, $rowcount, $sort, FALSE, TRUE);
-    }
-    else {
-      return CRM_Core_DAO::singleValueQuery("SELECT contact_id FROM {$this->_tableName}");
```

Comment:
See - this returns the results of the query - not the sql itself... and if there is more than one row it ??? - which seems invalid

Reply:
OK - see comment on main PR thread - I made sense of it